__tests__/Login.test.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import axios from 'axios';
+
+import { render, fireEvent, act, waitFor } from 'test-utils';
+import { Alert } from 'react-native';
+
+import LoginScreen from 'source/screens/LoginScreen';
+import MockNavigator from 'app/navigator/MockNavigator';
+
+import MockAdapter from 'axios-mock-adapter';
+
+jest.mock('jwt-decode', () => () => ({
+  exp: 123,
+}));
+
+jest.mock('react-native-network-info');
+
+jest.mock('../source/helpers/UrlHelper.js', () => ({
+  ...jest.requireActual('../source/helpers/UrlHelper.js'),
+  canOpenUrl: jest.fn().mockReturnValue(false),
+}));
+
+const mock = new MockAdapter(axios);
+
+mock.onPost('/auth/bankid/auth?apikey=').reply(200, {
+  success: true,
+  data: {
+    attributes: {
+      order_ref: 'theOrderRef',
+      auto_start_token: 'theAutoStartToken',
+    },
+  },
+});
+
+mock.onPost('/auth/bankid/collect?apikey=').reply(200, {
+  success: true,
+  data: {
+    attributes: {
+      completion_data: {
+        user: {
+          personal_number: '199803312389',
+        },
+      },
+    },
+  },
+});
+
+mock.onPost('/auth/token?apikey=').reply(200, {
+  success: true,
+  data: {
+    attributes: {
+      token: '123',
+    },
+  },
+});
+
```

Comment:
Maybe we also need a test for starting the login process? ie when a user starts the login and the bankid loading screens open? Or should that test be placed somewhere else?